ui/src/app/metadata/IMetadata.ts

```diff
@@ -0,0 +1,25 @@
+import {ApiJobCode} from "./job-codes/Jobcode"
+import {ApiNamedReference} from "./named-references/NamedReference"
+
+export interface IMetadata {
+  apiJobCodes?: ApiJobCode[],
+  apiNamedReferences?: ApiNamedReference[]
+}
+
+export class ApiMetadata implements IMetadata {
```

Comment:
Seems `IMetadata` and `ApiMetadata` are unused, but if there is plan to use them in the implementation ignore this comment.

Reply:
Yeah, I will rename the file to match with the only class used here, and let that code in this very file for future code inclusion.